src/recaptcha-wrapper.js

```diff
@@ -35,6 +35,28 @@ export function createRecaptcha () {
       this.assertLoaded()
       this.wait().then(() => window.grecaptcha.execute(widgetId))
     },
+	
+	loadRecaptcha () {
+		let recaptchaDetected = false;
+		let recaptchaSrc = 'https://www.google.com/recaptcha/api.js?onload=vueRecaptchaApiLoaded&render=explicit';
+		let documentScripts = document.getElementsByTagName('script');
+		if (documentScripts.length) {
+			for (let documentScript of documentScripts) {
+				if (documentScript.src === recaptchaSrc) {
+					recaptchaDetected = true;
+					return;
+				}
+			}
+		}
```

Comment:
This will scan through all script and check if recaptcha's script exists when each `vue-recaptcha` mounted.
Can we replace this with a function that only check at first execution?

Also I think we need to add a switch to turn off this feature.
Since somebody may want to custom the url to configure the language, etc. 
Or maybe perform a looser check.